_docs/architecture.md

```diff
@@ -135,3 +139,13 @@ If you have a balance with us you can use the `GET /v1/accounts` to get all or `
 If you use our API for collections, you can also use your internal balance as a wallet which would receive the funds collected from your customers.
 
 Please contact us to obtain our list of supported currency pairs for transactions.
+
+# API Limits
+
+We have soft limits around some of the endpoints that we ask customers to adhere to. These are the following:
+
+| Endpoint | Call limit / day | Notes |
```

Comment:
@CallumD please have a look if these look sensible. These should correspond to 1 transaction check per hour if you create 100 transactions a day, and one currency and account balance check every five minutes with a bit of leeway

Reply:
Yeah I think these are reasonable.